### ISSUE TITLE
Better document loadRemoteVersion

### ISSUE THREAD

**Issue by PatrickAlphaC** [ethereum/solc-js#583]
I'm trying to download versions of solidity using `loadRemoteVersion`, however, when I run:

```
solc.loadRemoteVersion("0.8.7", (resp) => {})
```
If I do a `console.log` where that method is defined, I get: 

https://binaries.soliditylang.org/bin/soljson-v0.8.10.js

Which is blank. However https://binaries.soliditylang.org/bin/soljson-latest.js isn't

Am I doing something wrong?

**Comment by PatrickAlphaC** [comment]
Basically, I'm having a hard time installing or working with a different version. I can do it find with `solc` and `solc-select` via the command line. 

But I'd love to be able to switch versions either in the command line _or_ in my javascript. 

Is this possible? Or do I have to write code to download different versions?

**Comment by PatrickAlphaC** [comment]
Or is the best way just to remove the package from node modules, change it in package-log.json, and then download the version I want?

**Comment by ekpyron** [comment]
You need to pass the long version string to ``loadRemoteVersion``, i.e.
```
solc.loadRemoteVersion("v0.8.7+commit.e28d00a7", (err, solc) => { console.log(err, solc); });
```
should work.
You can extract the long version string for each version from https://binaries.soliditylang.org/bin/list.json

**Comment by PatrickAlphaC** [comment]
Aha! I see! Thanks! 

Could I leave this issue open for the moment till we get a docs PR in?

**Comment by Elyx0** [comment]
Yes please, keep it open 🙏  that's secret knowledge otherwise

**Comment by PatrickAlphaC** [comment]
Yay open source :)

**Comment by alexreyes** [comment]
I believe this issue can be closed @ekpyron

**Comment by cameel** [comment]
True. @r0qs fixed it in #658. Closing.
